src/m_menu.c

```diff
@@ -3305,58 +3305,58 @@ void M_DrawWeapons(void)
 
 // Screen table definitions
 
-setup_menu_t stat_settings1[], stat_settings2[], stat_settings3[];
+setup_menu_t stat_settings1[], stat_settings2[];
 
 setup_menu_t* stat_settings[] =
 {
   stat_settings1,
   stat_settings2,
-  stat_settings3,
   NULL
 };
 
 enum {
   stat1_title1,
   stat1_rednum,
   stat1_graypcnt,
-  stat1_keys,
   stat1_stub1,
   stat1_title2,
+  stat1_coords,
   stat1_stats,
   stat1_time,
-  stat1_healthr,
-  stat1_healthy,
-  stat1_healthg,
-  stat1_armorr,
-  stat1_armory,
-  stat1_armorg,
-  stat1_ammor,
-  stat1_ammoy,
+  stat1_stub2,
+  stat1_title3,
+  stat1_backpack,
+  stat1_armortype,
+  stat1_smooth,
 };
 
-setup_menu_t stat_settings1[] =  // Status Bar and HUD Settings screen       
+static const char *show_widgets_strings[] = {
+    "OFF", "ON AUTOMAP", "ALWAYS", NULL
+};
+
+setup_menu_t stat_settings1[] =  // Status Bar and HUD Settings screen
 {
-  {"STATUS BAR"        ,S_SKIP|S_TITLE,m_null,M_X,M_Y},
+  {"STATUS BAR"        ,S_SKIP|S_TITLE,m_null,M_X,M_Y+stat1_title1*M_SPC},
 
   {"USE STANDARD COLORS",S_YESNO|S_COSMETIC,m_null,M_X,M_Y+stat1_rednum*M_SPC,   {"sts_always_red"}},
   {"GRAY %"            ,S_YESNO|S_COSMETIC,m_null,M_X,M_Y+stat1_graypcnt*M_SPC, {"sts_pct_always_gray"}},
```

Comment:
I always use colored numbers and gray %. 😎 

"Backpack changes Threasholds" was probably a bit more self-explaining when the thresholds were actually exposed in the menu. 😁 Say, you have 50 shells, then your ammo is maxed out, so the ammo widget is colored in green. Now you collect a backpack, and the maximum value increases to 100. Suddenly, you only carry 50% of the available ammo. Should your widget color now remain green or turn yellow?

Reply:
"BACKPACK SHIFTS AMMO COLOR"?

Reply:
Okay. I think we can safely cut the menus down to at least DSDA-Doom level, I've never seen any complaints about that.

Reply:
I'd like to get rid of the chat strings menu and the compatibility menu. But regarding the latter, I am not sure how to handle the Vanilla overflow emulations, then. I'd expect them to be all enabled, others have asked my repeatedly (in Crispy) to disable at least INTERCEPTS overflow emulation.

Reply:
The INTERCEPTS overflow is annoying during normal gameplay on some maps, but we set mbf21 complevel by default so it's not a big problem. I think we can remove it and see if anyone notices.

Reply:
I think we can only safely remove the Compatibility menu if we have automatic per-level-checksum applying of compatibility flags?

Reply:
> I think we can only safely remove the Compatibility menu if we have automatic per-level-checksum applying of compatibility flags?

Not really, we can tell users to play older maps with complevel vanilla. It would be nice to have automatic compatibility for popular maps though.